### ISSUE TITLE
arm64 windows not supported

### ISSUE THREAD

**Issue by loongs-zhang** [rust-lang/stacker#99]

**Comment by nagisa** [comment]
Correct, but more generally `psm` does not support windows at all and thus `stacker` uses fibers to implement segmented stacks...

Is that not working on arm64 windows somehow?

**Comment by jheuel** [comment]
We see [this](https://github.com/leptos-rs/cargo-leptos/actions/runs/13457858276/job/37605754318) in our release builds which are instrumented via cargo-dist.
